hydra-node/test/Hydra/Model.hs

```diff
@@ -605,6 +634,16 @@ performCommit parties party paymentUTxO = do
   makeAddressFromSigningKey :: CardanoSigningKey -> AddressInEra
   makeAddressFromSigningKey = mkVkAddress testNetworkId . getVerificationKey . signingKey
 
+toTxOuts :: [(CardanoSigningKey, Value)] -> [TxOut CtxUTxO]
+toTxOuts payments =
+  uncurry mkTxOut <$> payments
+ where
+  mkTxOut (CardanoSigningKey sk) val =
+    TxOut (mkVkAddress testNetworkId (getVerificationKey sk)) val TxOutDatumNone ReferenceScriptNone
+
+-- | NOTE: This function generates input 'Ix' that start from zero so
+-- can't be used with certainty when want to check equality with some 'UTxO'
+-- even though the value and the key would match.
```

Comment:
Could: fix this function to do that

Would be an alternative to `toTxOuts` + the sorting. A nice property here would be:

```
propIsDistributive :: Arbitrary a => (a -> b) -> [a] -> [a] -> Property
propIsDistributive fn as bs =
  fn as <> fn bs === fn (as <> bs)
```

with the test being `prop "realUTxO is distributive" $ propIsDistributive toRealUTxO`

(I hope this was distributivity .. long time no math)